www/_includes/layouts/components/menu.njk

```diff
@@ -1,95 +1,93 @@
-<nav class="menu">
-  <ol>
-    <li>
-      <span class="menu-section-header">Concepts</span>
-      <ol>
+<nav class="snow-toc">
```

Comment:
+1 for the `grid` -> snow changes! Grid is old leftover terminology from when we used CSS grid to layout the entire page, but then saw terrible slowdown on Safari